src/tests/unit/tests/DetailsView/components/reflow-assessment-view.test.tsx

```diff
@@ -4,25 +4,63 @@ import { AssessmentTestResult } from 'common/assessment/assessment-test-result';
 import { AssessmentData } from 'common/types/store-data/assessment-result-data';
 import { shallow } from 'enzyme';
 import * as React from 'react';
-import { Mock } from 'typemoq';
+import { IMock, Mock, Times } from 'typemoq';
 
+import { AssessmentViewUpdateHandler } from 'DetailsView/components/assessment-view-update-handler';
 import {
     ReflowAssessmentView,
     ReflowAssessmentViewDeps,
     ReflowAssessmentViewProps,
 } from '../../../../../DetailsView/components/reflow-assessment-view';
 
 describe('AssessmentViewTest', () => {
+    let updateHandlerMock: IMock<AssessmentViewUpdateHandler>;
+
+    beforeEach(() => {
+        updateHandlerMock = Mock.ofType<AssessmentViewUpdateHandler>();
+    });
+
     test('render for requirement', () => {
         const props = generateProps('requirement');
         const rendered = shallow(<ReflowAssessmentView {...props} />);
-        expect(rendered.getElement()).toMatchSnapshot();
+        expect(rendered.debug()).toMatchSnapshot();
```

Comment:
Any particular reason for this change?

Reply:
getElement was causing it to break, it seems like jest has issues serializing mocks for snapshot comparisons

Reply:
Ah yes! This is a known short-coming of typemoq with proxies. The easiest way to get around it would be to use:

`Mock.ofType(AssessmentViewUpdateHandler)` instead of what is currently at line 20. It's a bit more complicated if it's just an interface though ... ideally you should just want to _not_ pass in any mocks that are depending on interfaces as that'll cause the issue.

Reply:
Oh good to know! I'll change that then.